dotnet3.5/S3/CreateBucket/CreateBucket.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+// snippet-start:[S3.dotnet35.CreateBucket]
+using Amazon;
+using Amazon.S3;
+using Amazon.S3.Model;
+using Amazon.S3.Util;
+using System;
+using System.Threading.Tasks;
+
+namespace CreateBucket
+{
+    public class CreateBucket
+    {
+        // Specify your bucket region (an example region is shown).
+        private static readonly RegionEndpoint BUCKET_REGION = RegionEndpoint.USWest2;
+        private static IAmazonS3 _s3Client;
+
+        // Specify the name of the new bucket
+        private const string NEW_BUCKET_NAME = "*** provide the name of the bucket with source object ***";
+
+        static void Main()
+        {
+            _s3Client = new AmazonS3Client(BUCKET_REGION);
+            Console.WriteLine($"\nCreating a new bucket, named: {NEW_BUCKET_NAME}.");
+
+            CreatingBucketAsync(_s3Client, NEW_BUCKET_NAME).Wait();
+
+            // If FindBucketLocationAsyn doesn't return a value, the creation failed.
```

Comment:
Not sure why this comment is needed here. Isn't that what the try/catch in the called function does, stopping program execution if there's an error?

Reply:
Removed